### ISSUE TITLE
Experimenting and finding the best model for data using jupyter notebook

### ISSUE THREAD

**Issue by shivamshinde123** [shivamshinde123/Prediction-of-LC50-value-using-Quantitative-structure-activity-relationship-models---QSAR-models#4]

**Comment by shivamshinde123** [comment]
**Observations**
- Both machine learning models and neural network models tried
- Found the best accuracy model 
- Even the highest accuracy model is giving the r2 score of about 0.63 +/- 0.08. This is probably because of the limited amount of training data.
